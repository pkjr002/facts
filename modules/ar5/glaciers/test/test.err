Traceback (most recent call last):
  File "ar5_preprocess_glaciers.py", line 109, in <module>
    ar5_preprocess_glaciers(args.scenario, args.baseyear, args.tlm_data, args.pipeline_id, args.climate_data_file)
  File "ar5_preprocess_glaciers.py", line 27, in ar5_preprocess_glaciers
    tlm_dict = Import2lmData("surface_temperature", scenario, indir, refyear_start=1986, refyear_end=2005, climate_fname=fname)
NameError: name 'fname' is not defined
Traceback (most recent call last):
  File "ar5_fit_glaciers.py", line 87, in <module>
    ar5_fit_glaciers(args.gmip, args.pipeline_id)
  File "ar5_fit_glaciers.py", line 20, in ar5_fit_glaciers
    my_config = pickle.load(f)
UnboundLocalError: local variable 'f' referenced before assignment
Traceback (most recent call last):
  File "ar5_project_glaciers_dev.py", line 259, in <module>
    ar5_project_glaciers(args.seed, args.pyear_start, args.pyear_end, args.pyear_step, args.nmsamps, args.ntsamps, args.nsamps, args.pipeline_id)
  File "ar5_project_glaciers_dev.py", line 38, in ar5_project_glaciers
    my_data = pickle.load(f)
UnboundLocalError: local variable 'f' referenced before assignment
ls: cannot access glaciers-ar5-glaciers-ssp585*.nc: No such file or directory
mv: cannot stat ‘glaciers-ar5-glaciers-ssp585*.nc’: No such file or directory
