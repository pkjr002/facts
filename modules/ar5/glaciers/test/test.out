Initiating test in /scratch/rk509/test.1666837015...

Extracting data files...

Executing workflow...
python ar5_preprocess_glaciers.py --pipeline_id glaciers-ar5-glaciers-ssp585 --scenario ssp585 --baseyear 2005 --tlm_data 1 --climate_data_file test/input/fair-temperature-fair-ssp585_climate.nc
python ar5_fit_glaciers.py --pipeline_id glaciers-ar5-glaciers-ssp585 --gmip 2
Cannot open data file

python ar5_project_glaciers_dev.py --pipeline_id glaciers-ar5-glaciers-ssp585 --nsamps 500 --pyear_start 2020 --pyear_end 2300 --pyear_step 10 --seed 4321
Cannot open data file

python ar5_postprocess_glaciers_dev.py --pipeline_id glaciers-ar5-glaciers-ssp585 --chunksize 662 --locationfile /home/rk509/facts/modules/ar5/glaciers/test/../../../../scripts/moduletest/location_list.lst
Cannot open projfile


Collecting output files...

Skipping clean up...
